Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoOutput.kt

```diff
@@ -1,15 +1,14 @@
 package de.rki.coronawarnapp.ccl.dccwalletinfo.model
 
 import com.fasterxml.jackson.annotation.JsonIgnore
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.annotation.JsonSubTypes
 import com.fasterxml.jackson.annotation.JsonTypeInfo
 import com.fasterxml.jackson.annotation.JsonValue
+import com.fasterxml.jackson.databind.node.ObjectNode
 import de.rki.coronawarnapp.util.HashExtensions.toSHA256
 import org.joda.time.Instant
 
-@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Enabled to all see https://github.com/corona-warn-app/cwa-app-android/pull/4787/files#diff-a690de1cad015b4ed224878104d4eab09ae0e40573028390b3cd82880aa92791R52

Reply:
Removing this will result in a very fragile setup. It crashes if an unexpected attribute appears, and the the config is downloaded